src/components/Editor/SessionList.vue

```diff
@@ -132,6 +138,14 @@ export default {
 			return this.participantsWithoutCurrent.slice(0, 3)
 		},
 	},
+	methods: {
+		applyFocus() {
+			this.$refs.form.focus()
+		},
+		revertFocus() {
+			this.$editor.chain().focus().run()
```

Comment:
since there is only a single command in your chain i think you could do
```suggestion
			this.$editor.commands.focus()
```

Reply:
https://tiptap.dev/api/commands#execute-a-command

Reply:
Thanks!